public/data/guptasajal411.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Sajal Gupta",
+  "bio": "Full Stack Developer // CS Sophomore from India",
+  "avatar": "https://github.com/guptasajal411.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "http://github.com/guptasajal411",
```

Comment:
```suggestion
      "url": "https://github.com/guptasajal411",
```

Reply:
would it make a difference @Panquesito7 ?? if it stays http ??

Reply:
HTTPS is more securer than HTTP. It will redirect to HTTPS, but it's still better, IMO.